langchain/package.json

```diff
@@ -46,9 +50,9 @@
     "precommit": "tsc --noEmit && lint-staged",
     "clean": "rm -rf dist/ && node create-entrypoints.js clean",
     "prepack": "yarn build",
-    "test": "yarn clean && NODE_OPTIONS=--experimental-vm-modules jest --testPathIgnorePatterns=\\.int\\.test.ts",
```

Comment:
This isn't necessary anymore, and would eg. make the examples complain about failed imports after running the tests